spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -16,14 +16,16 @@
 
 package org.springframework.cloud.gcp.pubsub.core;
 
-import org.springframework.messaging.Message;
+import java.util.Map;
+
 import org.springframework.util.concurrent.ListenableFuture;
 
 /**
  * @author Vinicius Carvalho
+ * @author João André Martins
  */
 public interface PubSubOperations {
 
-	ListenableFuture<String> send(String topic, Message message);
+	ListenableFuture<String> publish(String topic, String payload, Map<String, String> headers);
```

Comment:
This is interesting. In the other PR you have open, I was suggesting using `PubsubMessage`, but maybe we can have both.